SAMtags.tex

```diff
@@ -251,21 +255,18 @@ \subsection{Barcodes}
 Same encoding as {\sf QUAL}, i.e., Phred score + 33.
 In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with spaces (`{\tt \textvisiblespace}') between the different strings from the same template. 
 
-\item[RX:Z:\tagvalue{sequence+}]
-Sequence bases from the unique molecular identifier. 
-These could be either corrected or uncorrected. Unlike {\tt MI}, the value may be non-unique in the file. 
-Should be comprised of a sequence of bases. 
-In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes with a hyphen (`{\tt -}') between the different barcodes.
+\item[CB:Z:\tagvalue{str}]
+Cell identifier based on the cellular-barcode sequence ({\tt CR}), with a correction optionally applied. The sequence may optionally be followed by a dash (`{\tt -}') and one or more alphanumeric characters to form an identifier. All reads with the same {\tt CB} tag putatively derive from the same cell.
+In the case of the cell identifier being based on multiple cellular-barcode sequences the recommended implementation concatenates all the barcodes with a hyphen (`{\tt -}') between the different barcodes.
 
-If the bases represent corrected bases, the original sequence can be stored in {\tt OX} (similar to {\tt OQ} storing the original qualities of bases.)
+\item[CR:Z:\tagvalue{str}]
+Cellular barcode. Sequence bases of the cellular-barcode as reported by the sequencer (uncorrected) with the respective quality scores stored in {\tt CY}. All reads with the same {\tt CR} tag putatively derive from the same cell. 
 
-\item[QX:Z:\tagvalue{qualities+}] 
-Phred quality of the unique molecular identifier sequence in the {\tt RX} tag. 
+\item[CY:Z:\tagvalue{qualities+}] 
+Phred quality of the cellular-barcode sequence in the {\tt CR} tag. 
```

Comment:
the barcode might be in the CB, so should end with "...{\tt CR} or {\tt CB} tag."

Reply:
As you well pointed out before, "This should only refer to the CR tag, since the CB tag contains an identifier and possibly corrected sequence for which these are not the qualities....", hence it makes sense that CB is not explicitly mentioned in CY.

Reply:
I agree with Nuno here - it should be CR only.  CB may also happen to be the same as CR, but let's not muddy the waters by linking CY to CR directly instead of via the correct indirect relationship.